### PR TITLE
Friendly message

### PR DESCRIPTION
**Pull request by owennw**
This provides a basic placeholder until we have a design.

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
I will test now and merge if all is good.

**Comment by bjedrzejewski** [comment]
@owennw waiting for updated design, then we merge after testing.

**Comment by owennw** [comment]
This is ready now.

File: src/services/quandl-service.js [hunk 1]
```diff
@@ -55,17 +41,37 @@
...
+                        method: 'GET',
+                        cache: true,
+                        transformResponse: function(data, headers) {
+                            var json = angular.fromJson(data);
+                            return filterByDate(json);
+                        }
+                    }
+                };
+
+                httpConfig.get.timeout = deferred.promise;
```
Comment (bjedrzejewski) [inline src/services/quandl-service.js]: Why is that and what is that?

Comment (bjedrzejewski) [inline src/services/quandl-service.js]: Can we write at least a short comment here? Is this related to friendly message?

**Comment by bjedrzejewski** [comment]
@owennw message still visible on hiding search

File: src/sidebars/search/search-controller.js [hunk 2]
```diff
@@ -39,6 +37,13 @@
                                 self.stocks.push(stock);
                             }
                         });
+
+                        var resultsCheck = $interval(function() {
+                            if (lastSearch.$resolved) {
+                                self.noResults = self.stocks.length === 0;
+                                $interval.cancel(resultsCheck);
+                            }
+                        }, 50);
```
Comment (bjedrzejewski) [inline src/sidebars/search/search-controller.js]: What is this logic doing?

**Comment by bjedrzejewski** [comment]
@owennw Cancel unwanted search should be a separate PR, it looks quite important and not necessary like the best thing to do.

**Comment by bjedrzejewski** [comment]
@owennw I looked on stack http://stackoverflow.com/questions/22915513/cancelling-http-request-angularjs and I found this comment which sums up what I think:

I think your approach is incorrect, you shouldn't be spamming your server and cancelling requests whenever the input changes. Instead, do a timeout and send a request when the user stops typing (e.g. whenever the input changes, set a 200ms timeout to contact the server, whenever the input changes clear the timeout and create a new one) – this is how it's usually done

**Comment by owennw** [comment]
That's just, like, your opinion, man.

But really -- the 200ms timeout is a tad arbitrary, and still results in http requests being created, and the results being discarded.

In any case I've removed it from this PR and used a second callback instead.

**Comment by bjedrzejewski** [comment]
It's not just an opinion. If you want to do it properly, you don't spam the server and cancel gets. That's just bad design.
